department.rb

```diff
@@ -0,0 +1,48 @@
+require './employee.rb'
+
+
+class Department
+
+  def initialize(department_name)
+    @department_name = department_name
+    @employees = []
+  end
+
+  def department_name
```

Comment:
Cool.  I'm glad that you know you need these methods.  You can also simplify your code by writing `attr_reader :department_name` and `attr_reader :employees` to have Ruby generate these methods for you automatically.